src/charm.py

```diff
@@ -108,16 +123,28 @@ def _hydra_layer(self) -> Layer:
         }
         return Layer(layer_config)
 
+    @property
+    def _hydra_service_is_running(self) -> bool:
+        try:
+            self._container.get_service(self._container_name)
```

Comment:
`get_service` won't tell you that the service is running but that it exists. you will probably want to have:
`service = self._container.get_service(self._container_name)` and instead of returning `True`, returning `service.is_running()`

Reply:
Wrong naming, I just want to check if the service is present, not that it's running. Renamed to `_hydra_service_is_created`